doc/source/whatsnew/v0.17.0.txt

```diff
@@ -180,6 +180,26 @@ Other enhancements
    s.drop_duplicates(keep=False)
 
 
+- ``GroupBy`` objects now have a ``pipe`` method, similar to the one on ``DataFrame`` and ``Series`` that allow for functions that take a ``GroupBy`` to be composed in a clean, readable syntax.
+
+Imagine that one has a function f that both takes and returns a ``DataFrameGroupBy``, a function g that takes a ``DataFrameGroupBy`` and a ``DataFrame``, and a function h that takes a ``DataFrame`` and returns a number.  Imagine further that each of these takes a single argument.
```

Comment:
This is just an extension, don't need this example; instead put a ref to the actual docs